### PR TITLE
Add a Python builder

### PR DESCRIPTION
**Pull request by colinking**
This PR adds support for a Python builder. It supports the latest Python 3 and can build any Python project with a `requirements.txt`.

I also included some improvements:
- If a build fails, all other builds will still be completed
- Build output is logged in a tabular format, so it is easier to identify which builds failed and why
- Adds a CODEOWNERS file

### CODE REVIEW

File: src/buildpack.ts [hunk 1]
```diff
@@ -53,6 +60,24 @@ export async function getDockerfile(b: Builder): Promise<string> {
...
       ENTRYPOINT ["deno", "run", "-A", "${b.builderConfig.entrypoint}"]
     `;
+  } else if (b.builder === "python") {
+    const requirementsPath = await find("requirements.txt", dirname(b.builderConfig.entrypoint));
+    if (!requirementsPath) {
+      throw new Error('Unable to find a requirements.txt')
+    }
+
+    contents = `
+      FROM python:3
```
Comment (joshma) [inline src/buildpack.ts:70]: Can we explicitly pin the minor version as well, and the Debian release? I'd suggest pinning it to `3.9-buster`. (We might have to make this configureable later either way.)

File: src/main.ts [hunk 2]
```diff
@@ -83,18 +87,64 @@ async function main() {
...
+          status: "fulfilled",
+          value: build,
+        })
+      } catch (err) {
+        results.push({
+          status: "rejected",
+          reason: {
+            build,
+            err,
+          },
```
Comment (joshma) [inline src/main.ts:116]: This is different from the parallel case, is that intentional? Missing status/value/reason?

**Review by joshma — APPROVED** [review]

File: src/buildpack.ts (hunk 1 above)
Comment (colinking) [reply src/buildpack.ts:70]: Ah, good point!

File: src/main.ts (hunk 2 above)
Comment (colinking) [reply src/main.ts:116]: Yeah, this is the way Promise.allSettled works: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled#using_promise.allsettled

Comment (joshma) [reply src/main.ts:116]: Ah woops, yeah that makes sense.

**Pull request merged**